tests/test_data/workspace.json

```diff
@@ -0,0 +1,4423 @@
+{
```

Comment:
Please make this file smaller to be easier to check the structure. Does it contain all possible weird structures we had issues with? (e.g. empty dict, empty lists, lists in dicts etc.)

Reply:
I don't see a reason to check the "weird" structures, as this script has nothing to do with that. It does not edit rest of the json. If addjson has "wierd" structures, they will be added. That would require testing `nodeJSON[keyJSON] = includeJSON` on line 30. I don't believe it's necessary.

Reply:
While adding json, it goes though all workspace. I would like to know if it will not fail when reaching some empty or none key value etc.

Reply:
added `null`, `{}`, and `[]` to workspace.json

Reply:
Cannot see that :(

Reply:
it got deleted when I changed the workspace. it's back now